### PR TITLE
More 93K comment cleanup

### PR DESCRIPTION
**Pull request by chrisnappi**
Add --no_inline_comments option (can also use $_testers_no_inline_comments
Fix cycle/vector count issue with expanded vectors
Add repeat comment for expanded vectors
Remove duplicate comments for expanded vectors
Clean up 93K comment formatting

### CODE REVIEW

File: approved/v93k/mm_dual.avc [hunk 1]
```diff
@@ -27,16 +27,16 @@ FORMAT TCLK TDI TDO TMS;
...
# ## end
# ###################################################################### ;
-R64                      nvmbist                    1 X X X   ;
+R1                       nvmbist                    0 X X X  
64.times do
$dut.pin(:tclk).drive!(0)
$dut.pin(:tclk).drive!(1)
end

;
```
Comment (ginty) [inline approved/v93k/mm_dual.avc]: Can we preserve the indentation?

Comment (chrisnappi) [inline approved/v93k/mm_dual.avc]: Will try - but I may need to eat one space.

Comment (chrisnappi) [inline approved/v93k/mm_dual.avc]: Okay, preserved spacing for those types of comments.

**Comment by ginty** [comment]
LGTM :+1:

**Comment by ginty** [comment]
Yeah, you're supposed to merge, then release

**Comment by chrisnappi** [comment]
Yeah, I keep thinking 'origen rc tag' should do a ' 'git fetch upstream' for me.  Sorry - fixing.

**Pull request merged**